setup/gcp/installer/util/utils.py

```diff
@@ -313,24 +314,34 @@ def sanitize_conf_values(conf_values):
     return conf_values
 
 
-def show_loading(loading_time, message='', max_number_of_dots=15):
+def show_loading(max_loading_time, exit_condition=None,
+                 message='', max_number_of_dots=15):
     """Show loading message, append dots to the end of the message up to
     a certain number of dots and repeat.
 
     Args:
-        loading_time (int): Loading time in seconds.
+        max_loading_time (int): Loading time in seconds.
+        exit_condition (func): Exit condition, a function that returns
```

Comment:
This would be more understand if the name is `exit_condition_checker`, since a checker is what we are passing in, not a condition.  And this usage would be easier to read:
`
 if exit_condition_checker and exit_condition_checker():
`

Reply:
Updated.